### PR TITLE
docs - add info about the advanced password check module

### PR DESCRIPTION
**Pull request by lisakowen**
add some docs about the advanced password check module.

doc review site link:  http://docs-lisa-gp-advpwdchk.cfapps.io/7-0/ref_guide/modules/adv_passwd_check.html

### CODE REVIEW

**Review by HaozhouWang — APPROVED** [review]

File: gpdb-doc/dita/ref_guide/modules/adv_passwd_check.xml [hunk 1]
```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE dita PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
+<dita>
+  <topic id="topic_fstrm">
+    <title>advanced_password_check</title>
+    <body>
+      <p>The <codeph>advanced_passwd_check</codeph> module provides password
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/modules/adv_passwd_check.xml:7]: inconsistency here between `passwd` followed by `password` in the actual module name.

File: gpdb-doc/dita/ref_guide/modules/adv_passwd_check.xml [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+          <codeph>shared_preload_libraries</codeph> in <codeph>postgresql.conf</codeph>:
+          <codeblock>shared_preload_libraries = '<i>other_libraries</i>,advanced_password_check'
+</codeblock>
+          and then restarting the Greenplum Database server.</p>
+      </body>
+    </topic>
+    <topic id="topic_using">
+      <title>Using the advanced_password_check Module</title>
+      <body>
+        <p><codeph>advanced_passwd_check</codeph> is a Greenplum Database module
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/modules/adv_passwd_check.xml:33]: `passwd` again here.

File: gpdb-doc/dita/ref_guide/modules/adv_passwd_check.xml [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+          that you can enable and configure to check password strings against
+          one or more user-defined policies. You can configure policies that:
+          <ul>
+            <li>Set a minimum password string length.</li>
+            <li>Set a maximum password string lentth.</li>
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/modules/adv_passwd_check.xml:38]: Typo: lentth

File: gpdb-doc/dita/ref_guide/modules/adv_passwd_check.xml [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+            <li>Define a custom ilst of special characters.</li>
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/modules/adv_passwd_check.xml:39]: ilst -> list

**Review by dyozie — CHANGES_REQUESTED** [review]

File: gpdb-doc/dita/ref_guide/modules/adv_passwd_check.xml (hunk 4 above)
Comment (lisakowen) [reply gpdb-doc/dita/ref_guide/modules/adv_passwd_check.xml]: thanks for catching those, david

**Review by dyozie — APPROVED** [review]

**Pull request merged**